daattali/beautiful-jekyll
# Update gh-pages to version 163

## Why?
the build should be done with a newer gh-pages to enable some additional features...

## Test Scenario
- remove the `#` in line  `gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw]` if you run jekyll with windows
  - For more details see https://github.com/tzinfo/tzinfo/wiki/Resolving-TZInfo::DataSourceNotFound-Errors
-  run `bundle install`
- Test build with `bundle exec jekyll serve`
- open http://127.0.0.1:4000

## Blocks ⬆️ 
- #261
